bridges/NineGagBridge.php

```diff
@@ -37,59 +37,66 @@ class NineGagBridge extends BridgeAbstract {
 				'name' => 'Section',
 				'type' => 'list',
 				'values' => array(
-					'Animals' => 'cute',
+					'Among Us' => 'among-us',
+					'Animals' => 'animals',
 					'Anime & Manga' => 'anime-manga',
+					'Anime Waifu' => 'animewaifu',
+					'Anime Wallpaper' => 'animewallpaper',
+					'Apex Legends' => 'apexlegends',
 					'Ask 9GAG' => 'ask9gag',
 					'Awesome' => 'awesome',
-					'Basketball' => 'basketball',
 					'Car' => 'car',
-					'Classical Art Memes' => 'classicalartmemes',
-					'Comic' => 'comic',
+					'Comic & Webtoon' => 'comic-webtoon',
+					'Coronavirus ' => 'coronavirus',
 					'Cosplay' => 'cosplay',
-					'Countryballs' => 'country',
-					'DIY & Crafts' => 'imadedis',
-					'Drawing & Illustration' => 'drawing',
-					'Fan Art' => 'animefanart',
-					'Food & Drinks' => 'food',
```

Comment:
When some list item for certain value becomes deleted, feed links applied patch become invalid.

For example: `?action=display&bridge=NineGag&context=Sections&g=food&t=hot&p=1&format=Html` is valid before applying this patch. After applying becomes invalid

Reply:
The site links to "https://9gag.com/food-drinks" (you can check that by searching for the category on the left hand side of the site). It looks like "https://9gag.com/food" was the old category, as it still reroutes to "https://9gag.com/food-drinks" if you type it in your browser.

The section was renamed from "Food" to "Food & Drinks" and 9gag seems to have done that multiple times (the whole drawing - diy - crafts sections for example).

So yes, you will need to update the feed link with the new category from 9gag.